easybuild/easyconfigs/f/findhap/findhap-4.eb

```diff
@@ -0,0 +1,38 @@
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'Tarball'
+
+name = 'findhap'
+version = '4'
+
+homepage = 'http://aipl.arsusda.gov/software/findhap/'
+description = """Find haplotypes and impute genotypes using multiple chip sets and sequence data"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['%(name)sV%(version)s.zip']
+source_urls = ['http://aipl.arsusda.gov/software/%(name)s/']
+
+modextrapaths = {
+    'PATH': '',
+}
+
+modloadmsg = """
+Copy the following file to the directory from which you will be running the executable: 
+\\$EBROOTFINDHAP/findhap.options
+
+Other files in the \\$EBROOTFINDHAP directory are:
+
+\\$EBROOTFINDHAP/genotypes.txt
+\\$EBROOTFINDHAP/chromosome.data
+\\$EBROOTFINDHAP/pedigree.file
+\\$EBROOTFINDHAP/sequences.readdepth
```

Comment:
@boegel It won't run without them, but the ones provided in the zip are just examples. What I've put in the message is really just a usage hint, though people wanting to run it should know already.